cmd/nerdctl/compose_exec_linux_test.go

```diff
@@ -207,3 +211,83 @@ services:
 	base.ComposeCmdWithHelper(unbuffer, "-f", comp.YAMLFullPath(), "exec", "-t=false", "svc0", "stty").AssertFail()                         // `-i`
 	base.ComposeCmdWithHelper(unbuffer, "-f", comp.YAMLFullPath(), "exec", "-i=false", "-t=false", "svc0", "stty").AssertFail()
 }
+
+func TestComposeExecWithIndex(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	base := testutil.NewBase(t)
+	var dockerComposeYAML = fmt.Sprintf(`
+version: '3.1'
+
+services:
+  svc0:
+    image: %s
+    command: "sleep infinity"
+    deploy:
+      replicas: 3
+`, testutil.CommonImage)
+
+	comp := testutil.NewComposeDir(t, dockerComposeYAML)
+	t.Cleanup(func() {
+		comp.CleanUp()
+	})
+	projectName := comp.ProjectName()
+	t.Logf("projectName=%q", projectName)
+
+	base.ComposeCmd("-f", comp.YAMLFullPath(), "up", "-d", "svc0").AssertOK()
+	t.Cleanup(func() {
+		base.ComposeCmd("-f", comp.YAMLFullPath(), "down", "-v").AssertOK()
+	})
+
+	// try random 5~10 times to ensure that results are stable
+	r, _ := rand.Int(rand.Reader, big.NewInt(5))
+	for i := 0; i < int(r.Int64()+5); i++ {
```

Comment:
can we just use a fixed number? e.g., 5 times or 10 times

Reply:
Done.